src/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -251,8 +252,16 @@
       <Link>Common\System\IO\TempFileCollection.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
     <Reference Include="System.Security.Cryptography.ProtectedData" />
+    <Reference Include="System.Security.Permissions" />
```

Comment:
I guess this reference is the reason @JeremyKuhne didn't add those APIs.  It does make me second guess the decision to add them. Given they are just virtual methods as opposed to abstract we could keep them excluded from the ref and anyone that targeted this contract would still successfully be able to run on desktop, we would just need to baseline them from an APICompat prospective. 

@JeremyKuhne do you have an opinion?